armi/plugins.py

```diff
@@ -665,6 +666,48 @@ def defineSystemBuilders() -> Dict[str, Callable[[str], "Composite"]]:
         and a ``"sfp"`` lookup, triggered to run after all other hooks have been run.
         """
 
+    @staticmethod
+    @HOOKSPEC(firstresult=True)
+    def getAxialExpansionChanger() -> type["AxialExpansionChanger"]:
+        """Produce the class responsible for performing thermal axial expansion.
+
+        Plugins can provide this hook to override or negate axial thermal expansion.
+        Will be used during initial construction of the core and assemblies, and
+        can be a standard class for performing all the thermally driven axial
+        expansion.
+
+        The first object returned that is not ``None`` will be used.
+        Plugins are encouraged to add the ``tryfirst=True`` arguments to their
+        ``HOOKIMPL`` invocations to make sure their specific are earlier in the
+        hook call sequence. The default reactor plugin
+        :class:`armi.reactor.ReactorPlugin` is marked ``trylast=True`` to help
+        put other implementations earlier in the sequence.
```

Comment:
Does this need to be updated given the App guidance you added in 9dbbca288fa024d87da69bd5731b58045ad9d174?

Reply:
I'm inclined to say no because those notes are a few lines lower in the `Notes` section of this docstring. If you think more docs are needed, or that paragraph needs to be higher up in the docs, let me know or make an in-line suggestion

Reply:
resolved with https://github.com/terrapower/armi/pull/1870/commits/bb01240d1324a01f94df41a2acff9c15b730bb2f